src/themes/uswitch/theme.json

```
@@ -2836,6 +2836,73 @@
               "mb": "md"
             }
           }
+        },
+        "featured": {
+          "itemsPerRow": 1,
+          "backgroundColor": "grey-05",
+          "display": "flex",
+          "flexDirection": ["column-reverse", null, "row-reverse"],
+          "justifyContent": ["none", null, "space-between"],
+          "&[class*='palette']": {
```

can we have it like this?
```suggestion
          "&.palette": {
```